src/components/spatial/Spatial.js

```diff
@@ -175,6 +175,10 @@ export default class Spatial extends AbstractSelectableComponent {
           colorValues,
           sliderValues,
           channelsOn,
+          onTileError: (err) => {
+            // eslint-disable-next-line no-console
+            console.log(err);
+          },
```

Comment:
Would it make sense to rethrow this error rather than logging? Not sure if you can.

```suggestion
          onTileError: (err) => {
            throw err;
          },
```

Reply:
I will try

Reply:
I think this will cause the application to crash, but could be wrong.

Reply:
Perhaps `console.warn` then? Trying to think of how to make the error message most visible (as a TileError)